frontend/src/hooks/domain/useSpaceAccess.ts

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react';
+import { authService } from '../../apis/services/auth.service';
+
+const useSpaceAccess = (hostId: number | undefined) => {
+  // TODO : 전역에서 내려주는 hostId로 변경
+  const [hasAccess, setHasAccess] = useState(false);
+  const [isLoadingAccess, setIsLoadingAccess] = useState(true);
+  useEffect(() => {
```

Comment:
가독성을 위해 한 줄 개행 부탁드려요🙏

```suggestion
  const [isLoadingAccess, setIsLoadingAccess] = useState(true);
  
  useEffect(() => {
```